### PR TITLE
HEEDLS-440 - refactored SectionContentService into SectionContentData…

### PR DESCRIPTION
**Pull request by DanBloxham-sw**
…Service

### JIRA link
https://softwiretech.atlassian.net/browse/HEEDLS-440

### Description
SectionContentService only contains methods fit for a data service, so it has been refactored into it.

### Screenshots
N/A

-----
### Developer checks
I have:
- [ ] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (controller, data services, services, view models etc) and manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme.
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [ ] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Data/Services/TutorialContentService.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ int tutorialId
                 // Using this list of other tutorials in the course we can work out if there is another item in the
```
Comment (stellake) [inline DigitalLearningSolutions.Data/Services/TutorialContentService.cs:45]: Should we also move this to a dataservice & rename?

Comment (DanBloxham-sw) [reply DigitalLearningSolutions.Data/Services/TutorialContentService.cs:45]: Yep. Was going to do them in separate MRs. Should I do it in this one instead?

Comment (stellake) [reply DigitalLearningSolutions.Data/Services/TutorialContentService.cs:45]: Feel free to do a separate one if that's the plan (I would have been happy either way) - will approve this 👍

**Review by stellake — APPROVED** [review]
Looks good! 👍

**Pull request merged**